ansible/collectory-standalone.yml

```diff
@@ -1,8 +1,8 @@
 - hosts: collectory
   roles:
-    - common
-    - java    
-    - mysql
-    - tomcat    
-    - apache
-    - collectory
+    - { role: common, collectory: ala-collectory, collectory_version: 1.0 }
+    - { role: java, collectory: ala-collectory, collectory_version: 1.0 }
+    - { role: mysql, collectory: ala-collectory, collectory_version: 1.0 }
+    - { role: tomcat, collectory: ala-collectory, collectory_version: 1.0 }
+    - { role: apache, collectory: ala-collectory, collectory_version: 1.0 }
+    - { role: collectory, collectory: ala-collectory, collectory_version: 1.0 }
```

Comment:
You only need to parameterise the collectory role. The others can remain as they were.